### ISSUE TITLE
inconsistant handling of date/time format

### ISSUE THREAD

**Issue by andreakb** [veraPDF/veraPDF-library#547]
I am using verapdf version 0.12.9 on a debian linux with java version 1.8.0_74, in NZST, and I am consistently getting this error:

```
<validationReport profile="PDF/A-1B validation profile" compliant="false">
    <statement>PDF file is not compliant with Validation Profile requirements.</statement>
    <details passedRules="101" failedRules="1" passedChecks="11707" failedChecks="1">
        <rule specification="ISO 19005-1:2005" clause="6.7.3" testNumber="1" status="failed" passedChecks="0" failedChecks="1">
            <description>If a document information dictionary does appear at a document, then all of its entries that have analogous properties in predefined XMP schemas, shall also be embedded in the file in XMP form with equivalent values.</description>
            <object>CosDocument</object>
            <test>doesInfoMatchXMP</test>
            <check status="failed">
                <context>root</context>
```

When I look at the binary of the pdf/a files I am checking, I see that the dates are the same. From one pdf/a I made on msword with a docx file in windows, the dates are: 
xmp:CreateDate2016-04-11T15:57:39+12:00/xmp:CreateDatexmp:ModifyDate2016-04-11T15:57:39+12:00/xmp:ModifyDate
and, from the info dictionary:
<</CreationDate(D:20160411155739+12'00') /ModDate(D:20160411155739+12'00') >>

on a pdf/a made from the same docx in libreoffice on linux, the dates are: 

from the info dictionary:
xmp:CreateDate2016-04-11T16:01:15+12:00/xmp:CreateDate
CreationDate(D:20160411160115+12'00')>>

However, verapdf is interpreting these dates differently. The xml output from the msword reports that the info dictionary dates are:
            <entry xmlns:ns2="http://www.verapdf.org/ValidationProfile" xmlns="" key="CreationDate">2016-04-11T15:57:39.000Z</entry>
            <entry xmlns:ns2="http://www.verapdf.org/ValidationProfile" xmlns="" key="ModDate">2016-04-11T15:57:39.000Z</entry>

And that the xmp dates as: 
 xmp:CreateDate2016-04-11T15:57:39+12:00/xmp:CreateDate
xmp:ModifyDate2016-04-11T15:57:39+12:00/xmp:ModifyDate

It looks like the UTC offset in the case of the information dictionary dates is not being  interpreted correctly. I am attaching both xml outputs and pdfs.

[docx libreoffice.pdf](https://github.com/veraPDF/veraPDF-library/files/212392/docx.libreoffice.pdf)
[docx msword.pdf](https://github.com/veraPDF/veraPDF-library/files/212393/docx.msword.pdf)
[docx libreoffice.txt](https://github.com/veraPDF/veraPDF-library/files/212396/docx.libreoffice.txt)
[docx msword.txt](https://github.com/veraPDF/veraPDF-library/files/212397/docx.msword.txt)

**Comment by THausherr** [comment]
After the "+12" to "+11" of both occurences in docx.libreoffice.pdf it works. PDFBox has the same bug (but I thought veraPDF is no longer using xmpbox?)
Might be related to this
https://issues.apache.org/jira/browse/PDFBOX-2420
although that one claims that +12 would be working.

**Comment by BezrukovM** [comment]
Fixed by veraPDF/veraPDF-pdfbox#57
